internal/servers/controller/handlers/managed_groups/validate_test.go

```diff
@@ -0,0 +1,141 @@
+package managed_groups
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/boundary/internal/auth"
+	"github.com/hashicorp/boundary/internal/auth/oidc"
+	pb "github.com/hashicorp/boundary/internal/gen/controller/api/resources/managedgroups"
+	pbs "github.com/hashicorp/boundary/internal/gen/controller/api/services"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/protobuf/types/known/fieldmaskpb"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+func fieldError(field, details string) string {
+	return fmt.Sprintf(`{name: %q, desc: %q}`, field, details)
+}
+
+func TestValidateCreateRequest(t *testing.T) {
```

Comment:
any reason why we shouldn't run the tests in this file in parallel?

Reply:
Probably not. I can update the account tests to do so as well.